tests/syft/lib/torch/tensor/tensor_remote_method_api_suite_test.py

```diff
@@ -146,32 +151,36 @@ def test_all_allowlisted_tensor_methods_work_remotely_on_all_types(
             valid_torch_command = False
         elif "takes 0 positional arguments but" in msg:
             valid_torch_command = False
+        elif "takes from 1 to 0 positional arguments but" in msg:
+            valid_torch_command = False
         elif "argument after * must be an iterable, not int" in msg:
             valid_torch_command = False
         elif "must be Number, not Tensor" in msg:
             valid_torch_command = False
         elif (
-            """TypeError("flatten(): argument 'start_dim' (position 1) must be int, not Tensor")"""
-            == msg
+            "TypeError" in msg
```

Comment:
Do we really need to check that if we already know the Exception is a TypeError?